### ISSUE TITLE
`hab studio run` broken on 0.62.1 on Mac OS

### ISSUE THREAD

**Issue by danielsdeleo** [habitat-sh/habitat#5625]
I first saw some breakage attempting to run our build, which does `hab studio run "source .studiorc; build components/automate-deployment"` While attempting to debug that, I see that I cannot even run basic commands via `hab studio run`:

```
hab studio run pwd
   hab-studio: Importing 'kallistec' secret origin key
» Importing origin key from standard input
★ Imported secret origin key kallistec-20180131222351.
   hab-studio: Importing 'kallistec' public origin key
» Importing origin key from standard input
★ Imported public origin key kallistec-20180131222351.
   hab-studio: Exported: HAB_AUTH_TOKEN=[redacted]
   hab-studio: Exported: HAB_ORIGIN=kallistec
   hab-studio: Exported: HAB_STUDIO_SUP=false
   hab-studio: Exported: GITHUB_TOKEN=[redacted]
/hab/pkgs/core/coreutils/8.25/20170513213226/bin/pwd: /hab/pkgs/core/coreutils/8.25/20170513213226/bin/pwd: cannot execute binary file
```

My docker daemon is acting up so I cannot get more environmental information just now, but it's an easy repro.

**Comment by smacfarlane** [comment]
This should be resolved in the 0.63.0 studio by #5559

**Comment by elliott-davis** [comment]
The fix will be released in 0.64.0
